app/upgrade_cudos.go

```diff
@@ -767,7 +767,7 @@ func withdrawGenesisStakingDelegations(genesisData *GenesisData, cudosCfg *Cudos
 		return fmt.Errorf("remaining bonded pool balance %s is too high", bondedPool.balance.String())
 	}
 
-	println("remaining bonded pool balance: ", bondedPool.balance.String())
+	app.Logger().Info("remaining bonded pool balance: ", bondedPool.balance.String())
```

Comment:
Please add `cudos merge: ` prefix for all log records like this:
```suggestion
	app.Logger().Info("cudos merge: remaining bonded pool balance: ", bondedPool.balance.String())
```